sdcm/cluster.py

```diff
@@ -2462,35 +2483,11 @@ def disable_client_encrypt(self):
             self.log.debug("Disabling client encryption on node")  # pylint: disable=no-member
             node.disable_client_encrypt()
 
-    def get_seed_nodes(self):
-        seed_nodes_ips = self.get_seed_nodes_from_node()
-        seed_nodes = []
-        for node in self.nodes:  # pylint: disable=no-member
-            LOGGER.debug("node: %s, seeds: %s", node.ip_address, seed_nodes_ips)
-            if node.ip_address in seed_nodes_ips:
-                node.is_seed = True  # TODO: check if we need to change the is_seed, later on
-                seed_nodes.append(node)
-        assert seed_nodes, "We should have at least one selected seed by now"
-        return seed_nodes
-
-    def get_seed_nodes_by_flag(self):
-        """
-        We set is_seed at two point, before and after node_setup.
-        However, we can only call this function when the flag is set.
-        """
-
-        node_ips = [node.ip_address for node
-                    in self.nodes if node.is_seed]  # pylint: disable=no-member
-        seeds = ",".join(node_ips)
-
-        assert seeds, "We should have at least one selected seed by now"
-        return seeds
-
     def _update_db_binary(self, new_scylla_bin, node_list):
         self.log.debug('User requested to update DB binary...')  # pylint: disable=no-member
 
         seed_nodes = self.get_seed_nodes()
-        non_seed_nodes = [n for n in self.nodes if not n in seed_nodes]  # pylint: disable=no-member
+        non_seed_nodes = self.get_non_seed_nodes()
```

Comment:
when you will convert to property no need to have this local variable

Reply:
done